### PR TITLE
Add notifications and response status to events

### PR DESCRIPTION
**Pull request by rodolfo3**

### CODE REVIEW

File: gcp_pilot/calendar.py [hunk 1]
```diff
@@ -14,11 +14,13 @@
 class Attendee:
     email: str
     optional: bool = False
+    response_status: str = 'needsAction'
```
Comment (lucasgomide) [inline gcp_pilot/calendar.py:17]: Could you describe into the commit message here why this is needed?

Comment (rodolfo3) [reply gcp_pilot/calendar.py:17]: It is the same as the `optional`: it is the default for this field: https://googleapis.github.io/google-api-python-client/docs/dyn/calendar_v3.events.html#patch

File: gcp_pilot/calendar.py [hunk 2]
```diff
@@ -171,6 +171,7 @@ def create_or_update_event(
         recurrence_amount: str = None,
         calendar_id: str = "primary",
         color: Color = None,
+        send_notifications: str = None
```
Comment (joaodaher) [inline gcp_pilot/calendar.py:174]: It's a boolean, right?
https://developers.google.com/calendar/api/v3/reference/events/insert

File: gcp_pilot/calendar.py [hunk 3]
```diff
@@ -188,6 +189,7 @@ def _build_time_field(dt):
             "description": description,
             "start": _build_time_field(dt=start_at),
             "end": _build_time_field(dt=end_at),
+            "sendNotifications": send_notifications,
```
Comment (joaodaher) [inline gcp_pilot/calendar.py:192]: It seems to be deprecated, they recommend `sendUpdates`, which is a string.
In order to support that, we should add the choices as a Enum to crate an easier interface.

**Review by lucasgomide — APPROVED** [review]

**Pull request merged**